WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -2803,13 +2803,17 @@ extension WooAnalyticsEvent {
         enum Keys: String {
             case path
             case entityName = "entity"
+            case debugPath = "debug_path"
```

Comment:
super nit: maybe `debugDecodingPath` and `debugDecodingDescription` could be better names for they not to be confused with the current `path` key.

Reply:
Good idea! Updated those name in 2f7c462.